api/core/signing.py

```diff
@@ -3,6 +3,8 @@
 
 
 def sign_payload(payload: str, key: str):
+    # TestWebHook on the frontend replicates this exact function, change the
+    # function there if this changes.
```

Comment:
1. For consistency's sake, let's call the frontend function `signPayload`.
2. I'd prefer to explicitly state the frontend module name in this here comment.

Reply:
@khvn26 I've addressed the comments and the pushed the fix

Reply:
Great, thanks! I've replicated your change across the rest of the new code comments.